wsmaster/che-core-api-auth-bitbucket/src/main/java/org/eclipse/che/security/oauth/BitbucketOAuthAuthenticator.java

```diff
@@ -50,35 +59,37 @@ public OAuthToken getToken(String userId) throws IOException {
     final OAuthToken token = super.getToken(userId);
     // Need to check if token is valid for requests, if valid - return it to caller.
     try {
-      if (token == null
-          || token.getToken() == null
-          || token.getToken().isEmpty()
-          || getJson(USER_URL, token.getToken(), BitbucketUser.class) == null) {
+      if (token == null || isNullOrEmpty(token.getToken())) {
         return null;
       }
+      testRequest(getTestRequestUrl(), token.getToken());
     } catch (OAuthAuthenticationException e) {
       return null;
     }
     return token;
   }
 
+  private String getTestRequestUrl() {
+    return bitbucketEndpoint.equals("https://bitbucket.org")
+        ? "https://api.bitbucket.org/2.0/user"
+        : bitbucketEndpoint + "/rest/api/1.0/application-properties";
+  }
```

Comment:
```suggestion
  private String getTestRequestUrl() {
    return "https://bitbucket.org".equals(bitbucketEndpoint)
        ? "https://api.bitbucket.org/2.0/user"
        : bitbucketEndpoint + "/rest/api/1.0/application-properties";
  }
```

Reply:
can we add some javadoc for that method?

Reply:
Added some javadoc to the `getTestRequestUrl()` method